RecoCTPPS/Configuration/python/RecoCTPPS_EventContent_cff.py

```diff
@@ -34,7 +34,8 @@
     'keep CTPPSPixelLocalTrackedmDetSetVector_ctppsPixelLocalTracks_*_*',
 
     # CTPPS common
-    'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
+    'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*',
+    'keep recoForwardProtons_ctppsProtons_*_*',
```

Comment:
should a similar be added in PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py so that the ctppsProtons are also available in miniAOD?